packages/expo-router/src/static/html.tsx

```diff
@@ -19,3 +22,16 @@ export function ScrollViewStyleReset() {
     />
   );
 }
+
+export function InnerRoot({
+  children,
+  loadedData,
+}: PropsWithChildren<{ loadedData: ServerDataLoaderData }>) {
+  const ref = React.createRef<ServerContainerRef>();
```

Comment:
I know this is just copied from previous implementation, but do we need it here?